aea/test_tools/acn_image.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""ACN Docker Image."""
```

Comment:
As mentioned - AEA is not allowed to contain this ACN specific stuff. That's not the concern of the AEA framework. I suggest you move everything ACN specific into a separate plugin or the conftest of the relevant package.

Reply:
Just move it into `test_libp2p`